crates/swc/src/lib.rs

```diff
@@ -851,24 +851,24 @@ impl Compiler {
         program: Option<Program>,
         handler: &Handler,
         opts: &Options,
-        custom_before_pass: impl FnOnce(&Program, &SingleThreadedComments) -> P1,
-        custom_after_pass: impl FnOnce(&Program, &SingleThreadedComments) -> P2,
+        comments: Option<&SingleThreadedComments>,
```

Comment:
Please make this non-optional, and take ownership

Reply:
Ok, modified